clients/java/signalr/.gitignore

```diff
@@ -7,5 +7,6 @@ bin/
 out/
 *.class
 *.jar
+*.pom
```

Comment:
Maybe we can just ignore the whole `build` folder?

Reply:
So we're already ignoring the entire build folder. The problem was that if you just run the gradle build task, the `signalr-client-0.1.0-preview1.pom` file would get created in the root of the java client. It was only getting moved by MSBuild. I updated the PR to have gradle move the .pom file to the build directory immediately after creating it, and then removed the move task that was in the MSBuild code.

Reply:
Can this be removed then?

Reply:
Ah, good. Sounds fine to me then. Thanks for clarifying!